src/main/java/core/basesyntax/operations/Return.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.operations;
+
+public interface Return {
```

Comment:
no. You should have one interface that describes some operations. And three implementations: BuyOperation, ReturnOperation, and SupplyOperation

Reply:
remove it